ui/panels-plugin/src/test/mock-query-result.ts

```diff
@@ -0,0 +1,1830 @@
+// Copyright 2022 The Perses Authors
```

Comment:
Holy crap this file 😂  I wonder if there's a shorthand way to generate this rather than a 1800 line file? Totally fine if not, but I saw this PR and was like "how did he write 2000 lines of tests?!"

Reply:
haha yeah good point, forgot to clean this up, just reduced that file to ~60 lines since most of the values weren't necessary for what I'm testing here